index.js

```diff
@@ -18,21 +18,21 @@ Pretty print a size from bytes
 */
 
 module.exports = function(size, nospace, one, places) {
```

Comment:
Could you also use: `module.exports = (size, nospace, one, places) => {`

Reply:
i'll do